metron-stellar/stellar-common/README.md

```diff
@@ -748,7 +728,7 @@ In the core language functions, we support basic functional programming primitiv
 ### `MULTISET_INIT`
   * Description: Creates an empty multiset, which is a map associating objects to their instance counts.
   * Input:
-    * input? - An initialization of the multiset
+    * input?? - An initialization of the multiset
```

Comment:
Why does this have a \??

Reply:
Gotcha, okay.  I think I would prefer something like (optional), what do you think?